frontend/src/metabase/redux/metadata.js

```diff
@@ -39,14 +39,11 @@ export const updateSegment = segment => {
   return Segments.actions.update(segment);
 };
 
-export const FETCH_REAL_DATABASES = Databases.actions.fetchList.toString();
```

Comment:
@ranquild we don't use those variables. What's the best way to get rid of them? should we replace them with strings or can I just delete them?